src/svg/community.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 20 17" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+<path d="M17.6933 2.30667C17.6944 2.8782 17.4827 3.42968 17.0994 3.85365C16.7161 4.27761 16.1887 4.54367 15.6199 4.6H15.3866H15.1466C14.703 4.55359 14.2824 4.37958 13.9356 4.09902C13.5889 3.81846 13.331 3.4434 13.193 3.01926C13.0551 2.59511 13.043 2.14006 13.1584 1.70922C13.2738 1.27838 13.5116 0.890232 13.843 0.591767C14.1744 0.293302 14.5853 0.0973262 15.0258 0.0275709C15.4664 -0.0421844 15.9177 0.0172726 16.3251 0.198742C16.7325 0.380212 17.0786 0.675911 17.3214 1.05003C17.5643 1.42415 17.6934 1.86065 17.6933 2.30667V2.30667Z" />
```

Comment:
Has this been put through a tool like https://jakearchibald.github.io/svgomg/ in order to reduce the precision of the paths while keeping the same visual precision?

Reply:
Now it has. Had forgotten the name of the tool so added it in a README in the same directory.